packages/cli/package.json

```
@@ -33,10 +33,8 @@
     "@types/node-fetch": "^2.5.5"
   },
   "scripts": {
-    "build": "yarn clean && yarn babel src --out-dir dist && cp -r ./src/commands/generate/templates/. ./dist/commands/generate/templates",
+    "build": "yarn cross-env NODE_ENV=production babel --delete-dir-on-start src --out-dir dist --copy-files --no-copy-ignored",
     "build:watch": "nodemon --ignore dist --exec 'yarn build'",
```

It might be cool to try the `--watch` flag in babel

```bash
  -w, --watch                                 Recompile files on changes.
```

```suggestion
    "build:watch": "yarn build --watch",
```

---

Then we could remove nodemon :)

---

Just more Babel grrrr regarding `—watch`. Couldn’t even tell what it was doing or if it was working… just acts like a hanging process.

Digging for more examples I found that it was working but there’s no cli output indication as such or which files have been changed. So you have to use `—verbose`. So I tried that. It helped and is more efficient because it can trigger granularly for specific files. But it wasn’t very convincing. Lastly, it will be a mess to implement along with the `.ts` build script, e.g. for “api/“ package. 

Definitely possible once #359 is merged and “build:ts” scripts are removed. But you should take it for a spin first. Just use `—watch —verbose`. I could be convinced it’s an overall improvement and gets rid of the nodemon package, but not sure it’s worth it at this point.